src/server/lib/middleware/errorHandler.ts

```diff
@@ -0,0 +1,36 @@
+import { NextFunction, Request } from 'express';
+import { ResponseWithLocals } from '@/server/models/Express';
+
+const appendQueryParameter = (url: string, parameters: string) => {
+  if (url.split('?').pop()?.includes(parameters)) {
+    return url;
+  }
+  if (url.includes('?')) {
+    return `${url}&${parameters}`;
+  } else {
+    return `${url}?${parameters}`;
+  }
+};
```

Comment:
this is fine for now, just making a note to see if this is something we can abstract to a library at some point